packages/dashboards/package-lock.json

```diff
@@ -1288,15 +1288,15 @@
     },
     "acorn-jsx": {
       "version": "3.0.1",
-      "resolved": "https://registry.npmjs.org/acorn-jsx/-/acorn-jsx-3.0.1.tgz",
+      "resolved": "http://registry.npmjs.org/acorn-jsx/-/acorn-jsx-3.0.1.tgz",
```

Comment:
Looks like the addon lock files are change from `https` to `http`. Can we revert this